recipes/pcl/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.10)
```

```suggestion
cmake_minimum_required(VERSION 3.1.3)
```

Don't need that version for these simple methods.